okta/resource_clients/policy_client.py

```diff
@@ -47,7 +49,7 @@ async def list_policies(
             [query_params.status] {str}
             [query_params.expand] {str}
         Returns:
-            list: Collection of Policy instances.
+            list: Collection of AuthorizationServerPolicy instances.
```

Comment:
Is this going to cause a breaking change?

Reply:
I guess, it might cause some breaking change, but all existing tests are passed, although we have no new tests. Probably, we can add tests with new PR, as it might require some time.

Reply:
There is an integration test which covers this change and it passed.